lib/logutils.py

```diff
@@ -20,28 +20,32 @@
 #  along with SmartHomeNG. If not, see <http://www.gnu.org/licenses/>.
 #########################################################################
 
+import logging
+
 """
 This module contains utils to be used in logging
 """
 
-class Filter(object):
+class Filter(logging.Filter):
     """
     This class builds a filter to be used in logging.yaml to configure logging
     
     Returning True tells logging to suppress this logentry,
     whereas False will include the record into further processing and eventual output
     """
     def __init__(self, name=''):
-        self.name = name
-        self.nlen = len(name)
+        if isinstance(name, list):
+            self.name = name
+        elif len(name) == 0:
+            self.name = []
+        else
+            self.name = [name]
```

Comment:
There seems to be a `:` missing after `else` (noticed in https://github.com/smarthomeNG/plugins/pull/163/commits/1a9fc54bd4efe0a8b6c50b5fd942192a9eff79dc#r219706643).